README.md

```diff
@@ -45,8 +45,10 @@ from connect.models.exception import FulfillmentFail, FulfillmentInquire, Skip
 
 class ExampleRequestProcessor(FulfillmentAutomation):
     def process_request(self, request):
+    
+        logger.info('Processing request {}'.format(request.id))
```

Comment:
please also include for product X, Contract ID and marketplace

Reply:
Fine, will do tomorrow.

Reply:
Done, I have added contract id and product and marketplace names (since I though it is more meaningful like this):

```python
logger.info('Processing request {} for contract {}, product {}, marketplace {}'
            .format(request.id,
                    request.contract.id,
                    request.asset.product.name,
                    request.marketplace.name))
```